### PR TITLE
Fix up the title

### PR DESCRIPTION
**Pull request by rgbkrk**

### CODE REVIEW

**Comment by rgbkrk** [comment]
Ref to https://github.com/rgbkrk/atom-script/commit/bdd71bb5009bf93f912f753975cad2c10d1463f6#commitcomment-15654540

@nixel2007

File: lib/view-runtime-observer.coffee [hunk 1]
```diff
@@ -20,7 +20,7 @@ class ViewRuntimeObserver
     @subscriptions.add runtime.onDidNotRun (ev) =>
       @view.showUnableToRunError ev.command
     @subscriptions.add runtime.onDidContextCreate (ev) =>
-      title = #{ev.lang} - #{ev.filename + (":#{ev.lineNumber}" if ev.lineNumber?)}
+      title = "#{ev.lang} - " + ev.filename + (if ev.lineNumber? then ":#{ev.lineNumber}" else "")
```
Comment (nixel2007) [inline lib/view-runtime-observer.coffee]: you shoud use `else ''` here (following the the style guide)

Comment (rgbkrk) [inline lib/view-runtime-observer.coffee]: Thanks!

**Comment by rgbkrk** [comment]
Thanks @nixel2007

**Comment by nixel2007** [comment]
@rgbkrk i'm glad to help :)

**Pull request merged**